packaging/linux/flatpak/dev.lizardbyte.sunshine.yml

```diff
@@ -208,3 +208,6 @@ modules:
       - sed -r -z -i -e
         's/("((do)|(undo)|(cmd)|(detached))"\s*:\s*\[?\n*\s*")(.*")/\1flatpak-spawn --host \7/gm'
         /app/share/sunshine/apps.json
+      - install -D packaging/linux/flatpak/install-mouse /app/bin/install-mouse
+      - install -D packaging/linux/flatpak/remove-mouse /app/bin/remove-mouse
```

Comment:
Can you add the `.sh` extension to these? I know it's not necessary to function but will help anyone using an IDE to develop.

If you add a directory like `packaging/linux/flatpak/scripts/`, could you just install the entire directory and not have a new line for each script? Or alternatively could it be a single script with arguments you can pass to it, like the AppRun for AppImage?